deployment/pypi/Makefile

```diff
@@ -41,7 +41,8 @@ build:
 	cp -r $(CWD)../../src/nni_manager/dist $(CWD)nni
 	cp -r $(CWD)../../src/nni_manager/config $(CWD)nni
 	cp -r $(CWD)../../src/webui/build $(CWD)nni/static
-	cp -r $(CWD)../../src/nasui/build $(CWD)nni/nasui
+    mkdir -p $(CWD)nni/nasui/build
+    cp -r $(CWD)../../src/nasui/build/. $(CWD)nni/nasui/build
```

Comment:
indent

Reply:
fixed, changed to use `Tab` instead of "space"